src/nginxconfig/generators/conf/drupal.conf.js

```diff
@@ -32,6 +34,15 @@ export default global => {
         deny: 'all',
     };
 
+    config['# Allow image styles to be handled by the CMS.'] = '';
```

Comment:
```suggestion
    config['# Drupal: allow image styles to be handled by the CMS'] = '';
```
Comment consistency in the final config :)